sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md

```diff
@@ -1,89 +0,0 @@
-# Generated code configuration
-
-Run `dotnet build /t:GenerateCode` to generate code.
-
-``` yaml
-azure-arm: true
-csharp: true
-library-name: Qumulo
-namespace: Azure.ResourceManager.Qumulo
-require: https://github.com/Azure/azure-rest-api-specs/blob/b73e2d320f1ae530ea5e78625dfe14a921dcf011/specification/liftrqumulo/resource-manager/readme.md
-output-folder: $(this-folder)/Generated
-clear-output-folder: true
-sample-gen:
-  output-folder: $(this-folder)/../samples/Generated
-  clear-output-folder: true
-skip-csproj: true
-modelerfour:
-  flatten-payloads: false
-use-model-reader-writer: true
-
-#mgmt-debug:
-#  show-serialized-names: true
-
-directive:
-  - from: Qumulo.Storage.json
-    where: $.definitions
-    transform: delete $.UserDetails.required
-
-format-by-name-rules:
-  'tenantId': 'uuid'
-  'ETag': 'etag'
-  'location': 'azure-location'
-  '*Uri': 'Uri'
-  '*Uris': 'Uri'
-
-acronym-mapping:
-  CPU: Cpu
-  CPUs: Cpus
-  Os: OS
-  Ip: IP
-  Ips: IPs|ips
-  ID: Id
-  IDs: Ids
-  VM: Vm
-  VMs: Vms
-  Vmos: VmOS
-  VMScaleSet: VmScaleSet
-  DNS: Dns
-  VPN: Vpn
-  NAT: Nat
-  WAN: Wan
-  Ipv4: IPv4|ipv4
-  Ipv6: IPv6|ipv6
-  Ipsec: IPsec|ipsec
-  SSO: Sso
-  URI: Uri
-  Etag: ETag|etag
-
-prepend-rp-prefix:
```

Comment:
There were a lot of renaming in `autorest.md`. Since you've deleted it, the class names are rolled back. You need to change back using `@@clientName`. For details, see https://azure.github.io/typespec-azure/docs/howtos/generate-client-libraries/02client/#renaming-the-client-name